test/Datadog.Trace.ClrProfiler.IntegrationTests/TestHelper.cs

```diff
@@ -30,41 +32,46 @@ protected TestHelper(string sampleAppName, ITestOutputHelper output)
 
         protected ITestOutputHelper Output { get; }
 
-        public string GetPlatform()
+        public static string GetPlatform()
         {
             return Environment.Is64BitProcess ? "x64" : "x86";
         }
 
-        public string GetOS()
+        public static string GetOS()
         {
             return Environment.OSVersion.Platform == PlatformID.Win32NT ? "win" :
                    Environment.OSVersion.Platform == PlatformID.Unix ? "linux" :
                    Environment.OSVersion.Platform == PlatformID.MacOSX ? "osx" :
                                                                           string.Empty;
         }
 
-        public string GetRuntimeIdentifier()
+        public static string GetRuntimeIdentifier()
         {
             return BuildParameters.CoreClr ? string.Empty : $"{GetOS()}-{GetPlatform()}";
         }
 
-        public string GetSolutionDirectory()
+        public static string GetSolutionDirectory()
         {
-            var pathParts = new Stack<string>(Environment.CurrentDirectory.ToLowerInvariant().Split(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar));
-            while (pathParts.Count > 0)
+            var directories = Environment.CurrentDirectory
```

Comment:
I think the previous approach working from the end of the path back to the root was safer, here it is going to give the wrong result if any parent directory is named `test`.

Reply:
Simplified the implementation to find `/test/` inside the string and getting a single substring instead of splitting into several segments and re-combining them.